dev/tasks/r/github.macos-linux.local.yml

```
@@ -58,6 +58,11 @@ jobs:
       - uses: r-lib/actions/setup-r@v2
         with:
           use-public-rspm: true
+      - name: Add R.framework/Resources/bin to the path
+        run: |
+          # CRAN builders have the entire bin here added to the path. This sometimes
+          # includes things like GNU libtool which name-collide with what we expect
+          echo "/Library/Frameworks/R.framework/Resources/bin" >> $GITHUB_PATH
```

How about adding a check whether `libtool` is GNU libtool here?

```suggestion
          echo "/Library/Frameworks/R.framework/Resources/bin" >> $GITHUB_PATH
      - name : Check whether libtool in R is used
        run: |
          if [ "$(which libtool)" != "/Library/Frameworks/R.framework/Resources/bin/libtool" ]; then
            echo "libtool provided by R isn't found: $(which libtool)"
            exit 1
          fi
```
```

---

Oh yes, thank you this is fantastic.